README.md

```diff
@@ -6,20 +6,22 @@ Please note that `teleporter-token-bridge` is still under active development and
 
 ## Overview
 
-Teleporter token bridge is an application that allows users to transfer tokens between Subnets. The bridge is a set of smart contracts that are deployed on both the source and destination Subnets, and leverages [Teleporter](https://github.com/ava-labs/teleporter) for cross-chain communication. The token bridges are designed to be permissionless: anyone can register compatible destination instances to allow for bridging tokens from the source chain to that new destination. The token bridge source on the home chain keeps track of token balances bridged to each destination instance, and handles returning the original tokens back to the user when bridged back to the home chain. Destination instances are registered with the source contract instance via Teleporter message upon creation.
+Teleporter token bridge is an application that allows users to transfer tokens between Subnets. The bridge is a set of smart contracts that are deployed across multiple Subnets, and leverages [Teleporter](https://github.com/ava-labs/teleporter) for cross-chain communication.
 
-The token bridge contracts take in either an ERC20 or native token to be bridged from a source chain, which can be referred to as the "home chain", and transfers the token to a destination chain to be represented as a new token. The new token representation on the destination chain can also either be an ERC20 or native token, allowing users to have any combination of ERC20 and native tokens between source and destination chains:
+Each bridge instance consists of one "hub" contract and at least one but possibly many "spoke" contracts. The hub contract lives on the Subnet where the asset to be bridge exists and locks that asset as collateral to be bridged to other Subnets. The spoke contracts, which each have a single specified hub contract, live on other Subnets that want to import the asset bridged by their specified hub. The token bridges are designed to be permissionless: anyone can register compatible spoke instances to allow for bridging tokens from the hub Subnet to that new spoke. The token bridge hub contract keeps track of token balances bridged to each spoke instance, and handles returning the original tokens back to the user when bridged back to the hub instance. Spoke instances are registered with their hub contract via Teleporter message upon creation.
```

Comment:
What do you think about capitalizing `Hub` and `Spoke` in our documentation?

Reply:
Interesting thought...I personally don't think it's necessary. "hub and spoke" is relatively common terminology and could be used as adjectives in this context (i.e. the hub chain), so I think it could cause more confusion/inconsistency than its worth.